File: mylibrary/src/main/java/com/alguojian/crash/CrashAdapter.java
package com.alguojian.crash;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.BaseViewHolder;

import java.text.SimpleDateFormat;
import java.util.Date;

/**
 * ${Descript}
 *
 * @author alguojian
 * @date 2018/8/18
 */
public class CrashAdapter extends BaseQuickAdapter<CrashBean, BaseViewHolder> {

    public CrashAdapter() {
        super(R.layout.item_crash);
    }

    @Override
    protected void convert(BaseViewHolder helper, CrashBean item) {

        helper.setText(R.id.title, item.crash.substring(0,50));
        helper.setText(R.id.time, new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(new Date(item.time)));
    }
}

File: logcrash/src/main/java/com/alguojian/logcrash/CrashAdapter.kt
package com.alguojian.logcrash

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.BaseViewHolder
import java.text.SimpleDateFormat
import java.util.*


class CrashAdapter : BaseQuickAdapter<CrashBean, BaseViewHolder>(R.layout.item_crash) {
    override fun convert(helper: BaseViewHolder, item: CrashBean) {
        if (item.getCrash().length >= 62) {
            helper.setText(R.id.title, item.getCrash().substring(0, 61))
        } else {
            helper.setText(R.id.title, item.getCrash())
        }
        helper.setText(R.id.time, SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(Date(item.time)))
    }
}